containers/podman-desktop
# fix: update the markdownlint targets

### What does this PR do?

Updates the "markdownlint" targets

### Screenshot / video of UI



### What issues does this PR fix or reference?

* #10482

### How to test this PR?

`pnpm markdownlint:check`


```
Finding: website/**/*.md !website/node_modules !website/api
Linting: 145 file(s)
Summary: 278 error(s)
Count File
    1 website/blog/2022-11-17-develop-podman-using-codespaces.md
    3 website/blog/2024-08-01-using-rhel-wsl-podman-machine.md
    4 website/blog/2024-08-08-release-1.12.md
   21 website/blog/2024-10-05-kubernetes-blog.md
    3 website/blog/2024-10-29-creating-an-extension.md
   34 website/blog/2024-10-31-release-1.14.md
    6 website/blog/2024-11-14-podman-desktop-cncf.md
   12 website/blog/2024-11-26-ai-lab-first-app.md
  105 website/blog/2024-12-12-release-1.15.md
    7 website/blog/2024-12-16-cncf-projects.md
    2 website/docs/discover-podman-desktop.md
    1 website/docs/extensions/index.md
    5 website/docs/extensions/install/using-extension.md
    3 website/docs/extensions/templates/index.md
    2 website/docs/installation/macos-install.md
    5 website/docs/installation/windows-install/index.md
    2 website/docs/kind/installing-extension.md
    5 website/docs/kubernetes/applying-a-yaml-manifest.md
    1 website/docs/kubernetes/configuring-editing-kube-object.md
    5 website/docs/kubernetes/deploying-a-pod-to-kubernetes.md
    3 website/docs/kubernetes/port-forwarding.md
    4 website/docs/migrating-from-docker/index.md
    8 website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md
    2 website/docs/minikube/installing-extension.md
    2 website/docs/podman/creating-a-podman-machine.md
    6 website/docs/podman/gpu.md
    1 website/docs/podman/podman-remote.md
    3 website/docs/troubleshooting/access-logs.md
    3 website/docs/troubleshooting/troubleshooting-extension-issues.md
    5 website/docs/troubleshooting/troubleshooting-podman-on-macos.md
    6 website/docs/troubleshooting/troubleshooting-podman-on-windows.md
    7 website/docs/troubleshooting/troubleshooting-podman.md
    1 website/tutorial/deploying-a-kubernetes-application.md
  278 [Total]
```
